pkg/archive/archive.go

```diff
@@ -212,3 +216,177 @@ func includeFile(fpath string) bool {
 	split := strings.Split(filepath.Clean(fpath), string(filepath.Separator))
 	return split[0] == config.InternalDir || split[0] == config.PublishDir || split[0] == "catalogs"
 }
+
+// Copied from mholt archiver repo. Temporary and can
+// add back to repo
+func Unarchive(a Archiver, source, destination string, excludePaths []string) error {
```

Comment:
Can you add comments above what you changed in the copied functions? They'll be easier to reference when rebasing later.